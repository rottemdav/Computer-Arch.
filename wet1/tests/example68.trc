2 1 14 2 local_history local_tables using_share_lsb
0x5084 T 0xc0cfc
0x901287c N 0xb6c474
0x8c9cc T 0x26fd0a4
0x8c9cc T 0x26fd0a4
0x901287c N 0xb6c474
0x2374 T 0x09c
0x9bb1cc T 0x92dcbf8
0x901287c T 0xb6c474
0xa2081168 T 0x2f0
0x560bb1d8 T 0x6225e1c
0xfcea8 T 0xd9f4b4
0x95bc4 N 0x1fc
0x505adc60 T 0x1a5cfc8
0x01f0d8 N 0x20633e98
0x01f0d8 T 0x20633e98
0xd44 N 0x3c68
0x9bb1cc T 0x92dcbf8
0x560bb1d8 N 0x6225e1c
0xfcea8 N 0xd9f4b4
0x3d0ab8ac N 0xca0c0
0x3d0ab8ac N 0xca0c0
0x9bb1cc T 0x92dcbf8
0x2374 T 0x09c
0x901287c T 0xb6c474
0xf00 N 0x6e44c
0x05424 T 0x2cd4
0x560bb1d8 N 0x6225e1c
0x5084 N 0xc0cfc
0x72f888 T 0xe1bc944
0xd8dac T 0x067c8
0x01f0d8 T 0x20633e98
0x01f0d8 T 0x20633e98
0x674f65c N 0xa238bd58
0x27c T 0x09cfa9e0
0x05424 T 0x2cd4
0x4ac0b50c T 0x84fb518
0x9bb1cc T 0x92dcbf8
0x8c9cc T 0x26fd0a4
0x8c9cc N 0x26fd0a4
0xd8dac T 0x067c8
0x505adc60 N 0x1a5cfc8
0x674f65c N 0xa238bd58
0xfcea8 T 0xd9f4b4
0xdb8e0 N 0x3b5408
0xb0d288 N 0x63baf98
0x2374 T 0x09c
0x8c9cc N 0x26fd0a4
0xd8dac T 0x067c8
0x05424 T 0x2cd4
0x901287c T 0xb6c474
0x2374 N 0x09c
0x901287c T 0xb6c474
0x71e272f8 N 0xc41bb80
0x5b9f3318 T 0xa7b26410
0x95bc4 N 0x1fc
0x2374 N 0x09c
0x05424 N 0x2cd4
0x674f65c N 0xa238bd58
0x0dca32b0 N 0x04c6c
0xb411cde8 T 0x1d2c
0x0dca32b0 T 0x04c6c
0x32e68 T 0x77c1864
0x5084 T 0xc0cfc
0x95bc4 N 0x1fc
0x901287c T 0xb6c474
0xdb8e0 T 0x3b5408
0x0dca32b0 N 0x04c6c
0xf00 T 0x6e44c
0xcea4 T 0xb4c
0x0d037a8 N 0xe48
0x4ac0b50c N 0x84fb518
0xd8dac N 0x067c8
0x3d0ab8ac N 0xca0c0
0x32e68 N 0x77c1864
0x72f888 T 0xe1bc944
0x674f65c T 0xa238bd58
0x8c9cc T 0x26fd0a4
0x0dca32b0 N 0x04c6c
0x27c T 0x09cfa9e0
0xd44 T 0x3c68
0x505adc60 T 0x1a5cfc8
0x8c9cc N 0x26fd0a4
0x5b9f3318 N 0xa7b26410
0xd44 N 0x3c68
0x65d8 N 0xe00
0x560bb1d8 T 0x6225e1c
0x0dca32b0 T 0x04c6c
0x5b9f3318 T 0xa7b26410
0x95bc4 T 0x1fc
0x901287c N 0xb6c474
0xd44 N 0x3c68
0x5084 T 0xc0cfc
0x505adc60 T 0x1a5cfc8
0x01f0d8 N 0x20633e98
0x901287c T 0xb6c474
0x0d037a8 T 0xe48
0x05424 N 0x2cd4
0x01f0d8 T 0x20633e98
0x32e68 T 0x77c1864
0x72f888 T 0xe1bc944
0x71e272f8 N 0xc41bb80
0x0dca32b0 T 0x04c6c
0x01f0d8 T 0x20633e98