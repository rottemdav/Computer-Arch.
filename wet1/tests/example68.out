0x5084 N 0x5088
0x901287c N 0x9012880
0x8c9cc N 0x8c9d0
0x8c9cc T 0x26fd0a4
0x901287c N 0x9012880
0x2374 N 0x2378
0x9bb1cc N 0x9bb1d0
0x901287c N 0x9012880
0xa2081168 N 0xa208116c
0x560bb1d8 N 0x560bb1dc
0xfcea8 N 0xfceac
0x95bc4 N 0x95bc8
0x505adc60 N 0x505adc64
0x1f0d8 N 0x1f0dc
0x1f0d8 N 0x1f0dc
0xd44 N 0xd48
0x9bb1cc N 0x9bb1d0
0x560bb1d8 N 0x560bb1dc
0xfcea8 N 0xfceac
0x3d0ab8ac N 0x3d0ab8b0
0x3d0ab8ac N 0x3d0ab8b0
0x9bb1cc N 0x9bb1d0
0x2374 N 0x2378
0x901287c N 0x9012880
0xf00 N 0xf04
0x5424 N 0x5428
0x560bb1d8 N 0x560bb1dc
0x5084 N 0x5088
0x72f888 N 0x72f88c
0xd8dac N 0xd8db0
0x1f0d8 N 0x1f0dc
0x1f0d8 T 0x20633e98
0x674f65c N 0x674f660
0x27c N 0x280
0x5424 N 0x5428
0x4ac0b50c N 0x4ac0b510
0x9bb1cc N 0x9bb1d0
0x8c9cc N 0x8c9d0
0x8c9cc T 0x26fd0a4
0xd8dac N 0xd8db0
0x505adc60 N 0x505adc64
0x674f65c N 0x674f660
0xfcea8 N 0xfceac
0xdb8e0 N 0xdb8e4
0xb0d288 N 0xb0d28c
0x2374 N 0x2378
0x8c9cc N 0x8c9d0
0xd8dac N 0xd8db0
0x5424 N 0x5428
0x901287c N 0x9012880
0x2374 N 0x2378
0x901287c N 0x9012880
0x71e272f8 N 0x71e272fc
0x5b9f3318 N 0x5b9f331c
0x95bc4 N 0x95bc8
0x2374 N 0x2378
0x5424 N 0x5428
0x674f65c N 0x674f660
0xdca32b0 N 0xdca32b4
0xb411cde8 N 0xb411cdec
0xdca32b0 N 0xdca32b4
0x32e68 N 0x32e6c
0x5084 N 0x5088
0x95bc4 N 0x95bc8
0x901287c N 0x9012880
0xdb8e0 N 0xdb8e4
0xdca32b0 N 0xdca32b4
0xf00 N 0xf04
0xcea4 N 0xcea8
0xd037a8 N 0xd037ac
0x4ac0b50c N 0x4ac0b510
0xd8dac N 0xd8db0
0x3d0ab8ac N 0x3d0ab8b0
0x32e68 N 0x32e6c
0x72f888 N 0x72f88c
0x674f65c N 0x674f660
0x8c9cc N 0x8c9d0
0xdca32b0 N 0xdca32b4
0x27c N 0x280
0xd44 N 0xd48
0x505adc60 N 0x505adc64
0x8c9cc N 0x8c9d0
0x5b9f3318 N 0x5b9f331c
0xd44 N 0xd48
0x65d8 N 0x65dc
0x560bb1d8 N 0x560bb1dc
0xdca32b0 N 0xdca32b4
0x5b9f3318 N 0x5b9f331c
0x95bc4 N 0x95bc8
0x901287c N 0x9012880
0xd44 N 0xd48
0x5084 N 0x5088
0x505adc60 N 0x505adc64
0x1f0d8 N 0x1f0dc
0x901287c N 0x9012880
0xd037a8 N 0xd037ac
0x5424 N 0x5428
0x1f0d8 N 0x1f0dc
0x32e68 N 0x32e6c
0x72f888 N 0x72f88c
0x71e272f8 N 0x71e272fc
0xdca32b0 N 0xdca32b4
0x1f0d8 N 0x1f0dc
flush_num: 59, br_num: 103, size: 100b
